### PR TITLE
fix: add requistes link to README

### PR DESCRIPTION
**Pull request by miaulalala**
Add a link to the installation prerequistes and add composer install to make command

Fixes https://github.com/nextcloud/calendar/issues/5247

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ If you find bugs or have problems with the CalDAV-Backend, you should ask the te
 
 ## Build the app
 
+Make sure you have all the [neccessary prereqisites](https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#prerequisites-for-manual-installation) for manuall installs
```
Comment (ChristophWurst) [inline README.md:50]:
```suggestion
Make sure you have all the [necessary prereqisites](https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#prerequisites-for-manual-installation) for manual installs
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
typos

File: README.md (hunk 1 above)
Comment (tcitworld) [reply README.md:50]: `prerequisites`

Comment (miaulalala) [reply README.md:50]: mondays

File: Makefile [hunk 2]
```diff
@@ -13,6 +13,12 @@ composer-init:
 	composer install --prefer-dist
 	composer update --prefer-dist
 
+composer:
+	composer install --prefer-dist
+
+composer-al:
```
Comment (ChristophWurst) [inline Makefile:19]: unused?

**Review by ChristophWurst — APPROVED** [review]

**Review by st3iny — APPROVED** [review]

**Review by tcitworld — APPROVED** [review]

**Pull request merged**